js/meshopt_clusterizer.module.d.ts

```diff
@@ -0,0 +1,35 @@
+// This file is part of meshoptimizer library and is distributed under the terms of MIT License.
+// Copyright (C) 2016-2024, by Arseny Kapoulkine (arseny.kapoulkine@gmail.com)
+
+export class Bounds {
```

Comment:
I would probably say this should use centerX etc. instead of 3-element arrays; we should probably also remove S8 data for now.

Reply:
done: [894160e](https://github.com/zeux/meshoptimizer/pull/738/commits/894160e14564daded8e460ab580b22ea7034da27)